administrator/components/com_admin/script.php

```diff
@@ -282,6 +284,48 @@ protected function uninstallExtensions()
         }
     }
 
+    /**
+     * This method is for migration for old updatenotification system plugin migration to task.
+     *
+     * @param   \stdClass  $data  Object with `extension_id` and `params` of the extension
+     *
+     * @return  void
+     *
+     * @since   __DEPLOY_VERSION__
+     */
+    private function migrateSessionGCPlugin($data)
+    {
+        if (!$data->enabled) {
+            return;
+        }
+
+        // Get the plugin parameters
+        $params = new Registry($data->params);
+
+        /** @var SchedulerComponent $component */
+        $component = Factory::getApplication()->bootComponent('com_scheduler');
+
+        /** @var TaskModel $model */
+        $model = $component->getMVCFactory()->createModel('Task', 'Administrator', ['ignore_request' => true]);
+        $task  = [
+            'title'           => 'SessionGC',
+            'type'            => 'session.datapurge',
```

Comment:
![image](https://github.com/joomla/joomla-cms/assets/66922325/5c1d76f2-21f9-4e21-afc0-5b11ba636df8)
https://github.com/joomla/joomla-cms/blob/bb83935e4389783491a9a7971c8ec61a54120e80/plugins/task/sessiongc/src/Extension/SessionGC.php#L51
```suggestion
            'type'            => 'session.gc',
```